DOL.WHD.Section14c.Web/src/modules/components/sectionAssurances/sectionAssurancesTemplate.html

```diff
@@ -19,41 +23,30 @@
       <li>Wages paid to all employees under FLSA section 14(c) will be adjusted at periodic intervals, at least once a year, to reflect changes in the prevailing wage paid to experienced workers, who do not have disabilities, employed in the vicinity for essentially the same type of work.</li>
     </ol>
   </div>
-  <div class="form-question-block" ng-class="validate('signature.agreement') ? 'usa-input-error margintop' : 'margintop'">
+  <div class="dol-form-question-block" ng-class="validate('signature.agreement') ? 'usa-input-error margintop' : 'margintop'">
     <span class="usa-input-error-message" role="alert" ng-show="validate('signature.agreement')">{{ validate('signature.agreement') }}</span>
-    <fieldset class="usa-fieldset-inputs form-question-answer">
-      <legend class="hide">Signature agreement</legend>
-      <ul class="usa-unstyled-list">
-        <li>
-          <input id="agreement" type="checkbox" name="agreement" ng-value="true" ng-model="formData.signature.agreement"></input>
-          <label for="agreement" class="full-width-label">I agree to use an electronic signature. By entering my Full Name and Title below, I certify that I am authorized to accept these representations and assurances on behalf of the organization named on this application.</label>
-        </li>
-      </ul>
-    </fieldset>
+    <ul class="usa-unstyled-list">
+      <li>
+        <input id="agreement" type="checkbox" name="agreement" ng-value="true" ng-model="formData.signature.agreement"></input>
+        <label for="agreement" class="full-width-label">I agree to use an electronic signature. By entering my Full Name and Title below, I certify that I am authorized to accept these representations and assurances on behalf of the organization named on this application.</label>
+      </li>
+    </ul>
```

Comment:
I'm curious what the `<fieldset><label>` stuff accomplished before.  I like the reduced code, though.  😛